src/lib/backend.ai-client-esm.ts

```diff
@@ -2018,7 +2018,10 @@ class ResourcePreset {
   /**
    * Return the GraphQL Promise object containing resource preset checking result.
    */
-  async check(param = null): Promise<any> {
+  async check(param: {group: string, scaling_group?: string}): Promise<any> {
+    if (param && (param.scaling_group === null || param.scaling_group === '')) {
+      delete param.scaling_group;
+    }
```

Comment:
I think this change is not necessary because:
- This type of preprocessing is better placed at the caller of this function. Additionally, there is already preprocessing here: https://github.com/lablup/backend.ai-webui/blob/9002a0ff96dec42e0ac9f085ea8400209929ce24/src/components/backend-ai-resource-broker.ts#L539-L553